server/zanata-frontend/src/frontend/package.json

```diff
@@ -138,6 +137,7 @@
     "redux-sac": "0.5.3",
     "redux-thunk": "1.0.3",
     "redux-watch": "1.1.1",
+    "rtl-detect": "^1.0.0",
```

Comment:
maybe someone more skilled than I can get this working?

Reply:
@aeng ^ re: our discussion today - will leave this PR so someone else can try and get the language detection working